template.yaml

```diff
@@ -52,14 +57,20 @@ Mappings:
       userProfileTableKeyArn: arn:aws:kms:eu-west-2:761723964695:key/79b55133-ed01-45d8-94cd-99be01711e2c
       txmaEventEncryptionKeyArn: arn:aws:kms:eu-west-2:816047645251:key/d74787b0-8d11-4dd9-8691-7c0e26856225
       orchToAuthTokenSigningKeyAlias: arn:aws:kms:eu-west-2:761723964695:key/f2b0090b-56aa-4ff1-80fd-e8f8334199a7
+      ipvAud: undefined
```

Comment:
Rather than adding another mapping value, you can calculate this and give `!Ref AWS::NoValue` for the relevant environments - you can take a look at how I've done it in my draft ipv callback pr 🙂